jinja2/utils.py

```diff
@@ -512,6 +511,8 @@ def __next__(self):
         self.pos = (self.pos + 1) % len(self.items)
         return rv
 
+    next = __next__
```

Comment:
In that case test will be failed, there are two iteration ways... I'm sorry, right now I can use only web for pull request and I do not know how to squash commits here.